content/components/navigation-links-list.html

```diff
@@ -57,6 +57,9 @@ <h4 class="va-nav-linkslist-title">{{ link.title }}</h4>
               </a>
             </li>
           {% endif %}
+          {% if link.sibling !== false %}
```

Comment:
What's this?

Reply:
This should be `{% if link.sibling %}`, and has been changed.